database/fingertips-db/PostDeployment/PopulateWeatherForecast.sql

```diff
@@ -1,5 +1,5 @@
 -- This file contains SQL statements that will be executed after the build script.
-
+TRUNCATE TABLE WeatherForecast;
```

Comment:
I added this as otherwise we were ending up with multiple sets of entries in the `WeatherForecast` table when running the .dacpac file against a DB that had already been populated.

Reply:
OK with this for now but in the longer term, for reference data we should probably put a guard in to figure out if we need to execute it or not.

Reply:
Yes, definitely!